ntoskrnl/include/internal/amd64/intrin_i.h

```diff
@@ -59,6 +59,54 @@ KiInitGdtEntry(PKGDTENTRY64 Entry, ULONG64 Base, ULONG Size, UCHAR Type, UCHAR D
     Entry->MustBeZero = 0;
 }
 
+extern ULONG64 KeFeatureBits;
+
+FORCEINLINE
+VOID
+KiSaveXState(
+    _Out_ PVOID Buffer,
+    _In_ ULONG64 ComponentMask)
+{
+    ULONG64 npxState = ComponentMask & ~2;
```

Comment:
I think it would be reasonable to use a `ULL` suffix here.

Reply:
Should use `~XSTATE_MASK_LEGACY_SSE`